editor/blocks/quote-block/index.js

```diff
@@ -0,0 +1,23 @@
+const { query, html, text } = wp.blocks.query;
+
+wp.blocks.registerBlock( 'wp/quote', {
+	title: 'Quote',
+	icon: 'format-quote',
+
+	attributes: {
+		value: query( 'blockquote', html( 'p' ) ),
+		citation: query( 'cite,footer', text() )
+	},
+
+	edit( attributes, onChange ) {
+		return wp.element.createElement( wp.blocks.EditableQuote, {
+			value: attributes.value,
+			citation: attributes.citation,
+			onChange: ( value ) => onChange( { value } )
+		} );
+	},
+
+	save( attributes ) {
+		return wp.element.createElement( 'p', null, attributes.value );
```

Comment:
Yep, A clone of the output element (could be different than the edit element shown on the editor).  I personally still have a preference for return a string instead of an element, but I'm ok trying this way and see how it feels, we can still reconsider later.